_posts/elements/2015-04-09-buttons.md

```diff
@@ -46,7 +46,6 @@ info: Buttons should be used for performing actions within the edX environment.
     <button type="button" class="btn btn-link btn-base">Link Base</button>
     <button type="button" class="btn btn-link btn-large">Link Large</button>
     <button type="button" class="btn btn-link btn-small">Link Small</button>
-    <button type="button" class="btn btn-link btn-base" disabled>Link Disabled</button>
```

Comment:
@cptvitamin We removed the disabled link, because links are either enabled and in the DOM or not in the DOM. Would you agree with this?